### PR TITLE
update to all latest versions per greenkeeper

### PR DESCRIPTION
**Pull request by maxbeatty**
I've been using rebass for a project, really like it, and want to keep it moving forward so tried to update all the dependencies. `npm test` passed locally for me but I'm sure there's more to test

[I left a comment on any open PR labeled greenkeeper that had a newer version PR](https://github.com/jxnblk/rebass/pulls?utf8=%E2%9C%93&q=is%3Aopen%20is%3Apr%20label%3Agreenkeeper%20comments%3A%3E0). Those should be safe to close.

This PR closes:

1. closes #197 
2. closes #195 
3. closes #176 
4. closes #181 
5. closes #204 
6. closes #198 
7. closes #189 
8. closes #188 
9. closes #187 
1. closes #199 
1. closes #180 
1. closes #174 
1. closes #167 
1. closes #208 
1. closes #207 
1. closes #206 
1. closes #209 
1. closes #212 
1. closes #213 
1. closes #214 
1. closes #217 
1. closes #218 
1. closes #219 
1. closes #220 
1. closes #222 
1. closes #223

### CODE REVIEW

File: test/CardImage.spec.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ describe('CardImage', () => {
   let tree
 
   beforeEach(() => {
-    renderer.render(<CardImage />)
+    renderer.render(<CardImage src='http://placehold.it/256' />)
```
Comment (maxbeatty) [inline test/CardImage.spec.js]: silences warning about `src` prop being required

File: test/Slider.spec.js [hunk 2]
```diff
@@ -69,7 +69,7 @@ describe('Slider', () => {
     })
 
     it('should have a background gradient', () => {
-      expect(input.props.style.backgroundImage).toMatch(/^linear\-gradient/)
+      expect(input.props.style.backgroundImage).toMatch(/^linear-gradient/)
```
Comment (maxbeatty) [inline test/Slider.spec.js]: `standard` identified as unnecessary escape

File: package.json [hunk 3]
```diff
@@ -21,42 +21,42 @@
...
     "react-geomicons": "^2.0.5",
-    "react-router": "^2.8.0",
+    "react-router": "^3.0.2",
     "reflexbox": "^2.1.0",
-    "standard": "^8.0.0",
+    "standard": "^8.6.0",
     "static-react": "^3.1.0",
-    "webpack": "^1.13.2",
-    "webpack-dev-server": "^1.15.1"
+    "webpack": "^1.14.0",
```
Comment (9renpoto) [inline package.json]: plz upgrade webpack v2

Comment (maxbeatty) [inline package.json]: I think that's better left for a separate PR. My goal was to get all the dependencies upgraded within their defined semver range and make as little code changes as possible in doing so

**Comment by jxnblk** [comment]
Thanks for this. I think green keeper was getting out of hand 😎

**Pull request merged**